addons/settings/fnc_readFile.sqf

```diff
@@ -0,0 +1,32 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_settings_fnc_readFile
+
+Description:
+    Reads and applies settings from a file.
+    Only considered as public API from a static settings addon!
```

Comment:
Sure, I just tried to somehow make it it clear it is semi-public because every with a static setting addon would call it for example.